manifests/install.pp

```diff
@@ -19,36 +19,36 @@
   }
 
   package { 'foreman-service':
-    ensure => installed,
+    ensure => $foreman::version,
   }
 
   if $foreman::dynflow_manage_services {
     package { 'foreman-dynflow-sidekiq':
-      ensure => installed,
+      ensure => $foreman::version,
     }
   }
 
   if $foreman::ipa_authentication and $foreman::ipa_manage_sssd {
     package { 'sssd-dbus':
-      ensure => installed,
+      ensure => $foreman::version,
     }
```

Comment:
This package is not a subpackage of `foreman` so it has its own versioning.

Reply:
Thanks, Ewoud. Indeed, this is a silly search&replace mistake. It's fixed now :)